Stable-Branch-Strategy.md

```diff
@@ -0,0 +1,103 @@
+branch is created (stable-1.3) starting from master and the older stable branch (stable-1.1)
+is dropped from maintenance.
+
+
+| Branch | Original version | New version |
+|--|--|--|
+| `master` | `1.3.0-rc1` | `1.3.0` |
+| `stable-1.3` | N/A| `1.3.0` |
+| `stable-1.2` | `1.2.1` | `1.2.2` |
+| `stable-1.1` | `1.1.3` | (unmaintained) |
+
+Note, the stable-1.1 branch will still exist with tag 1.1.3, but under current plans it would
+not be maintained further.
+   
+## Backporting Process 
+
+Development will occur against the master branch and applicable code commits should also be submitted
+against the stable branches. Some guidelines:
+ (1) Only bug and security fixes which do not introduce inter-component dependencies are
+ candidates for stable branches. These PRs should be marked with "bug" in GitHub.
+ (2) Once a PR is created against master which meets requirement of (1), a comparable one
+ should also be submitted against the stable branches. It is the responsibility of the submitter
+ to apply their pull request against stable, and it is the responsibility of the
+ reviewers to help identify stable-candidate pull requests.
+ 
+## Continuous Integration Testing
+
+The test repository will also be forked to create stable branches from master. Full CI
+will run on each stable and master PR, using its respective tests repository branch.
+
+### An alternative method for CI testing:
+
+Ideally the continuous integration infrastructure will run the same test suite on both master
+and the stable branches.  When tests are modified or new feature tests are introduced, explicit
+logic should exist within the testing CI to make sure only applicable tests are executed against
+stable and master. This is not in place today, but should be considered in the longer term.
+
+## Release Management
+
+### Patch releases
+
+Releases will be made on a weekly cadence for patch releases, which will include a GitHub release as
+well as binary packages. These patch releases will be made for both stable branches, and a 'release candidate'
+for next `MAJOR` or `MINOR` will be created from master. If there are no changes across all the repositories, no
+release will be created and an announcement will be made on the developer mailing list to highlight this.
+If a release is being made, each repository will be tagged for this release, regardless
+of whether changes have been introduced.
+
+## Minor releases
+
+###  Frequency
+Minor releases are less frequent in order to provide a more stable baseline for users. They are currently
+running on a six week cadence. As the Kata Containers code base matures it is expected this will become
+a much longer duration.
+
+### Compatibility
+Kata will guarantee compatibility between components that are within one minor release of each other. 
+ 
+This is especially critical for dependencies which cross between host (runtime, shim, proxy) and
```

Comment:
Lines 97-101 suggested rewrite: 

This is critical for dependencies that cross between host (runtime, shim, and proxy) and the guest (hypervisor, rootfs, and agent). For example, consider a cluster with a long-running deployment and workload-never-dies on kata version 1.1.3 components. If the operator updates the Kata components to the next new minor release (i.e. 1.2.0), we need to guarantee that the 1.2.0 runtime still communicates with the 1.1.3 agent within workload-never-dies.